packages/api/src/utils/createTask.ts

```diff
@@ -669,6 +670,20 @@ export const enqueueTriggerRuleJob = async (data: TriggerRuleJobData) => {
   })
 }
 
+export const enqueueWebhookJob = async (data: CallWebhookJobData) => {
+  const queue = await getBackendQueue()
+  if (!queue) {
+    return undefined
+  }
+
+  return queue.add(CALL_WEBHOOK_JOB_NAME, data, {
+    priority: 1,
```

Comment:
I think priority should be lower for this one. Probably 5 or 10

Reply:
yeah it makes sense.

i think we can set priority = 5 for this and trigger-rule jobs